spec/traits.dd

```diff
@@ -939,6 +940,9 @@ $(H2 $(LNAME2 specialkeywords, Special Keywords))
     $(P $(CODE __FILE__) and $(CODE __LINE__) expand to the source
```

Comment:
I meant that if you pass relative it expands to relative and vice versa...not well formulated. So don't bother and ignore this comment, unless you manage to explain accurately the behavior of this macro.